### DISCUSSION TITLE
Board: Resize board/background

### DISCUSSION THREAD

**Discussion by kanarelo** [pikasojs/pikaso#32] [Ideas]
As it is, once created, I am not sure if we can update the dimensions of the board.

We may need `editor.board.rescale()` to be allowed to take arguments.

This may affect the background, so, the background (`image` & `overlay`) should also be resized to the new dimensions. Although, the background should be stretched to fit into the new background.

- User should be able to choose whether to crop the background image, or stretch to fit/fill canvas.

**Comment by raminious** [comment]
this is more complicated. everything including created shapes and paths should recalculate. currently you can resize the board through `editor.board.stage`

**Reply by kanarelo** [reply]
I agree... I have had some time to think about this, and I think you are right... And it might introduce new bugs because everything will need to be recalculated.
